src/backend/src/game/game.controller.ts

```diff
@@ -22,43 +25,105 @@ export class GameController {
     private playersRepository: Repository<Player>,
     @InjectRepository(Round)
     private roundsRepository: Repository<Round>,
+    private roundService: RoundService,
+    private gameSseService: GameSseService,
+    private gameService: GameService,
   ) {}
 
+  @Get('sse')
+  async subscribePlayer(
+    @Body('playerId') playerId: number,
+    @Body('gameId') gameId: number,
+    @Res() res: Response,
+  ): Promise<void> {
+    res.set({
+      'Content-Type': 'text/event-stream',
+      'Cache-Control': 'no-cache',
+      'X-Accel-Buffering': 'no',
+      Connection: 'keep-alive',
+    });
+
+    await this.gameSseService.subscribeClient(res, {
+      playerId,
+      gameId,
+    });
+  }
```

Comment:
I just remembered that here we want add the playerId and GameId as query params instead of body elements (similar to waiting room controller)